content/shmem_finalize.tex

```diff
@@ -24,7 +24,7 @@
     releases resources used by the \openshmem library. This collective
     operation requires all \acp{PE} to participate in the call. There is an
     implicit global barrier in \FUNC{shmem\_finalize} so that pending
-    communications are completed, and no resources can be released until all
+    communications \newtext{by all threads of the \ac{PE}} are completed, and no resources can be released until all
```

Comment:
I fear there might have been a lot of conversation on this which I missed. 

I think "\newtext{by all threads of the \ac{PE}}" can be safely removed. We already say else where that actions initiated by a thread as if they are initiated by the PE.